frontend/src/api/tests/StreamApi.test.js

```diff
@@ -2,56 +2,81 @@ import axios from "axios";
 import Enzyme from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
 import { afterEach, describe, expect, jest, it } from "@jest/globals";
-// import StreamApi itself
 import * as StreamApi from "../StreamApi";
+import StreamInfo from "../../model/StreamInfo";
+import { convertToDataObject } from "../../model/ConvertDataFormat";
+import * as StreamFixture from "./StreamFixture";
 
 Enzyme.configure({ adapter: new Adapter() });
 
 jest.mock("axios");
 
-const mockSingleStream = {
-  streamId: 1,
-  streamName: "Test 1"
-};
-const mockStreams = [mockSingleStream, { streamId: 2, streamName: "Test 2" }];
-
-const respAllStreams = {
-  target: {
-    streams: mockStreams
-  }
-};
-const respSingleStream = {
-  target: {
-    stream: mockSingleStream
-  }
-};
+const { firstStreamResponse } = StreamFixture;
+const { secondStreamResponse } = StreamFixture;
+const { allStreamsResponse } = StreamFixture;
```

Comment:
nitpick: this could be one line
```suggestion
const { firstStreamResponse, secondStreamResponse, allStreamsResponse } = StreamFixture;
```